update readme (#49)
pharmaR/pharmapkgs: README.Rmd

options(
    repos = "https://raw.githubusercontent.com/pharmaR/pharmapkgs/refs/heads/main/inst/repos",
    pkgType = "source"
)
available_packages <- available.packages()
nrow(available_packages)
```

Once set, calling `available.packages()` will list the packages available in the pharmapkgs repository. The repository currently contains around 7,000 packages, a subset of CRAN's nearly 30,000 packages.

### Installation Limitations
The pharmapkgs repository differs from a traditional CRAN repository. It does not store package tarballs but instead relies on a `download_url` field that links to the CRAN mirror on GitHub. Base R's `install.packages()` expects package tarballs within the same repository, so it does not work natively with pharmapkgs. Instead, users should use **pak** to install packages from this repository.

## Installation via `pak`
```{r, eval=FALSE}
options(
    repos = "https://raw.githubusercontent.com/pharmaR/pharmapkgs/refs/heads/main/inst/repos",
    pkgType = "source"
)
install.packages("pak")
pak::pkg_install("<package_name>")
```
The `pak` package understands the `download_url` field and fetches the package source accordingly.

## Filtering Packages

> pharmapkgs enhances the standard `available.packages()` functionality by allowing users to filter packages based on additional metadata. Users can define custom filters for selecting packages.

Reviewer: The functionality to filter packages is already available on `available.package`. What the R repository adds is the information to be able to filter it according to the assessment done on them. 

```suggestion
pharmapkgs enhances the standard `PACKAGES` file by adding additional metadata. 
Users can define custom filters for selecting packages.
```
